packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue

```diff
@@ -377,7 +377,7 @@ function updateNodeLabel(id: string, label: string) {
 	const outputPort = cloneDeep(props.node.outputs?.find((port) => port.id === id));
 	if (!outputPort) return;
 	outputPort.label = label;
-	emit('update-output-port', outputPort);
+	emit('update-output', outputPort);
```

Comment:
I think @blanchco made the labels computed properties, so this function `updateNodeLabel` may not be needed anymore.

Reply:
Thank you for the catch, talked with @blanchco to confirm what I was doing.